packages/perseus/src/widgets/radio/__tests__/answerless-radio.test.tsx

```diff
@@ -0,0 +1,177 @@
+            advanceTimers: jest.advanceTimersByTime,
+        });
+
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+
+        // Mocked for loading graphie in svg-image
+        global.fetch = jest.fn(() =>
+            Promise.resolve({
+                text: () => "",
+                ok: true,
+            }),
+        ) as jest.Mock;
+    });
+
+    it("is interactive with full widget options", async () => {
+        // Arrange
+        const fullWidgetOptions: PerseusRadioWidgetOptions =
+            getRadioWidgetOptions();
+
+        const fullItem: PerseusRenderer = {
+            content: "[[☃ radio 1]]",
+            images: {},
+            widgets: {
+                "radio 1": {
+                    type: "radio",
+                    options: fullWidgetOptions,
+                },
+            },
+        };
+
+        // Act
+        const {renderer} = renderQuestion(fullItem);
+
+        await userEvent.click(
+            screen.getByRole("checkbox", {name: "(Choice A) Correct 1"}),
+        );
+        await userEvent.click(
+            screen.getByRole("checkbox", {name: "(Choice B) Correct 2"}),
+        );
+
+        // assert that functionality previous based on answers still works
+        expect(screen.getByRole("group", {name: "Choose 2 answers:"}));
+
+        const userInput = renderer.getUserInputMap();
+        const score = scorePerseusItem(fullItem, userInput, "en");
+
+        // Assert
+        expect(score).toHaveBeenAnsweredCorrectly();
+    });
+
+    it("is interactive with stripped widget options", async () => {
+        // Arrange
+        const fullWidgetOptions: PerseusRadioWidgetOptions =
+            getRadioWidgetOptions();
+
+        const fullItem: PerseusRenderer = {
+            content: "[[☃ radio 1]]",
+            images: {},
+            widgets: {
+                "radio 1": {
+                    type: "radio",
+                    options: fullWidgetOptions,
+                    version: {major: 2, minor: 0},
+                },
+            },
+        };
+
+        const strippedItem: PerseusRenderer = {
+            content: "[[☃ radio 1]]",
+            images: {},
+            widgets: {
+                "radio 1": {
+                    type: "radio",
+                    options: getRadioPublicWidgetOptions(fullWidgetOptions),
+                },
+            },
+        };
+
+        // Act
+        const {renderer} = renderQuestion(strippedItem);
+
+        await userEvent.click(
+            screen.getByRole("checkbox", {name: "(Choice A) Correct 1"}),
+        );
+        await userEvent.click(
+            screen.getByRole("checkbox", {name: "(Choice B) Correct 2"}),
+        );
+
+        // assert that functionality previous based on answers still works
+        expect(screen.getByRole("group", {name: "Choose 2 answers:"}));
+
+        const userInput = renderer.getUserInputMap();
```

Comment:
This is important because we can do everything we need to do pre-scoring with the answerless data now as far as I can tell.